xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/test/java/org/xwiki/refactoring/script/RequestFactoryTest.java

```diff
@@ -58,14 +59,16 @@
     @MockComponent
     private ModelContext modelContext;
 
-    private DocumentReference userReference = new DocumentReference("wiki", "Users", "Carol");
+    private DocumentReference userReference = new DocumentReference("wiki1", "Users", "Carol");
```

Comment:
Was changing `wiki` to `wiki1` needed?

Reply:
I wanted to ensure it was really this one which was used for the current wiki reference. "wiki" is the standard name for the main wiki, so it could have been another reference.

Reply:
The standard name fo the main wiki is `xwiki`, not `wiki`(that's why we use `wiki` in the tests actually)

Reply:
Note sure what you mean. XWiki#DEFAULT_MAIN_WIKI is `xwiki` but RequestFactoryTest does not seems to manipulate any API that would go trough this. I don't where where `wiki` could come from.

Reply:
My bad I could keep using wiki then.